src/scales/scale.category.js

```diff
@@ -63,17 +63,20 @@ module.exports = Scale.extend({
 		return me.ticks[index - me.minIndex];
 	},
 
-	// Used to get data value locations.  Value can either be an index or a numerical value
-	getPixelForValue: function(value, index, datasetIndex) {
+	_configure: function() {
 		var me = this;
 		var offset = me.options.offset;
 
-		// 1 is added because we need the length but we have the indexes
-		var offsetAmt = Math.max(me.maxIndex + 1 - me.minIndex - (offset ? 0 : 1), 1);
+		Scale.prototype._configure.call(me);
 
-		var isHorizontal = me.isHorizontal();
-		var valueDimension = (isHorizontal ? me.width : me.height) / offsetAmt;
-		var valueCategory, labels, idx, pixel;
+		me.start = me.minIndex - (offset ? 0.5 : 0);
```

Comment:
Do we want to add new public fields? Also, it's a little confusing that we'll have `start` and `_start`. If we need both, can we rename one to help differentiate?

Reply:
Many scales already have a public `start`, so figured it doesn't matter that much.
Could be `_startValue` or `_startIndex` (`_startValue` can be consistent, `_startIndex` would equal `_startTime` in time scale etc.), I just hate wasting bytes to property names :)

Reply:
Renamed both to make it clear